Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -84,12 +92,15 @@ - (void)cancel
     if (t) {
         [t cancel];
     }
+    _finished = YES;
```

Comment:
Oh I see, in my tiredness I got confused looking through the diff, I see its fine now